example/myapp/pubspec.lock

```diff
@@ -401,4 +408,4 @@ packages:
     source: hosted
     version: "3.1.0"
 sdks:
-  dart: ">=2.12.0 <3.0.0"
+  dart: ">=2.14.0 <3.0.0"
```

Comment:
Happens because I have latest stable installed (Flutter 2.5.3). Even though pubspec.yaml refer to `sdk: ">=2.12.0 <3.0.0"`. I think, if we commit a `pubspec.lock` file (As we currently do for the examples), then it should match the latest stable. We could also remove these `.lock` files from git altogether? What do you prefer?

Reply:
they were initially committed cause of the guidance that apps should commit these files.  If these lock files in the examples are getting in the way we could remove them. But lets wait for the next time it happens. I suggest we revert this for now

Reply:
I don't think the guidance is intended for examples like this. It makes sense to lock dependencies for an app released on an app store, but less so for example on github that people compile them self. I would like to remove these, as they will be in the way for someone on a never version of the tooling than us.

Reply:
Also.. for realm package we require `flutter: ">= 2.5.0"` which includes dart which includes 2.14. I think that is where the updated constraint stems from